### PR TITLE
Fix deanonymizer documentation

### PR DESCRIPTION
**Pull request by shiranr**

### CODE REVIEW

**Review by SharonHart — DISMISSED** [review]

File: docs/anonymizer/index.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ entities with desired values.
 
 Persidio anonymizer comes with predefined operators but can easily be extended.
 
+Presidio-Anonymizer supports both anonymization and deanonymization of encrypted entities.
```
Comment (omri374) [inline docs/anonymizer/index.md:8]: Consider creating a paragraph out of these three sentences, making the reading more fluent

File: docs/anonymizer/index.md [hunk 2]
```diff
@@ -70,6 +73,29 @@ Persidio anonymizer comes with predefined operators but can easily be extended.
     
     ```
 
+=== "Python"
```
Comment (omri374) [inline docs/anonymizer/index.md:76]:
```suggestion
```
Part of the previous Python tab

File: docs/anonymizer/index.md [hunk 3]
```diff
@@ -70,6 +73,29 @@ Persidio anonymizer comes with predefined operators but can easily be extended.
...
+
+    Deanonymized ecncrypted text:
```
Comment (omri374) [inline docs/anonymizer/index.md:78]:
```suggestion
    Deanonymize encrypted text:
```

File: docs/anonymizer/index.md [hunk 4]
```diff
@@ -70,6 +73,29 @@ Persidio anonymizer comes with predefined operators but can easily be extended.
...
+    ```python
+    from presidio_anonymizer import DeanonymizeEngine
+    from presidio_anonymizer.entities.engine import AnonymizerResult, OperatorConfig
+    
+    # Initialize the engine with logger.
+    engine = DeanonymizeEngine()
+    
+    # Class the deanonymize function with the text, anonymizer results and
+    # Operators to define the deanonymization type.
+    result = engine.deanonymize(
```
Comment (omri374) [inline docs/anonymizer/index.md:89]: Maybe pass the `result` you obtained on the previous code snippet?

File: presidio-anonymizer/README.MD [hunk 5]
```diff
@@ -80,6 +80,19 @@ My name is Inigo Montoya. You Killed my Father. Prepare to die. BTW my number is
     My name is Inigo Montoya. You Killed my Father. Prepare to die. BTW my number is: <
     PHONE_NUMBER><SSN>.
 
+### Deanonymize
+
+Presidio deanonymize currently only contains one operator:
```
Comment (omri374) [inline presidio-anonymizer/README.MD:85]:
```suggestion
Presidio deanonymize currently contains one operator:
```

File: presidio-anonymizer/README.MD [hunk 6]
```diff
@@ -80,6 +80,19 @@ My name is Inigo Montoya. You Killed my Father. Prepare to die. BTW my number is
     My name is Inigo Montoya. You Killed my Father. Prepare to die. BTW my number is: <
     PHONE_NUMBER><SSN>.
 
+### Deanonymize
```
Comment (omri374) [inline presidio-anonymizer/README.MD:83]: Maybe Deanonymizer? To have the same structure as the Anonymizer

File: presidio-anonymizer/README.MD [hunk 7]
```diff
@@ -112,6 +125,26 @@ print(result)
...
+from presidio_anonymizer import DeanonymizeEngine
+from presidio_anonymizer.entities.engine import AnonymizerResult, OperatorConfig
+
+# Initialize the engine with logger.
+engine = DeanonymizeEngine()
+
+# Class the deanonymize function with the text, anonymizer results and
+# Operators to define the deanonymization type.
+result = engine.deanonymize(
+    text="My name is S184CMt9Drj7QaKQ21JTrpYzghnboTF9pn/neN8JME0=",
```
Comment (omri374) [inline presidio-anonymizer/README.MD:139]: Same here, consider using the anonymizer results

File: docs/anonymizer/index.md (hunk 1 above)
Comment (shiranr) [reply docs/anonymizer/index.md:8]: Done.

File: presidio-anonymizer/README.MD (hunk 6 above)
Comment (shiranr) [reply presidio-anonymizer/README.MD:83]: Done.

File: docs/anonymizer/index.md [hunk 8]
```diff
@@ -1,9 +1,10 @@
 # Presidio Anonymizer
 
 The Presidio anonymizer is a Python based module for anonymizing detected PII text
-entities with desired values.
+entities with desired values. Presidio anonymizer supports both anonymization and 
+deanonymization by using operators. Operators are built-int text manipulation classes
```
Comment (omri374) [inline docs/anonymizer/index.md:5]:
```suggestion
deanonymization by using operators. Operators are built-in text manipulation classes 
```
nit: typo

File: docs/anonymizer/index.md [hunk 9]
```diff
@@ -70,6 +71,28 @@ Persidio anonymizer comes with predefined operators but can easily be extended.
...
+    Deanonymize an already encrypted text by the anonymizer:
+    
+    ```python
+    from presidio_anonymizer import DeanonymizeEngine
+    from presidio_anonymizer.entities.engine import AnonymizerResult, OperatorConfig
+    
+    # Initialize the engine with logger.
+    engine = DeanonymizeEngine()
+    
+    # Class the deanonymize function with the text, anonymizer results and
```
Comment (omri374) [inline docs/anonymizer/index.md]: Sentence isn't clear

Comment (shiranr) [reply docs/anonymizer/index.md]: Done.

**Review by omri374 — APPROVED** [review]

**Review by SharonHart — APPROVED** [review]

**Pull request merged**